### DISCUSSION TITLE
Welcome to exam-month5-redux Discussions!

### DISCUSSION THREAD

**Discussion by Tohirjon-Odilov** [Tohirjon-Odilov/exam-month5-redux#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tohirjon-Odilov** [comment]
Nima gaplar

**Reply by Tohirjon-Odilov** [reply]
### Bosvoliroradi

**Reply by 1lyosbek** [reply]
nima gap

**Reply by 1lyosbek** [reply]
tinch

**Comment by Tohirjon-Odilov** [comment]
Tinchmisilar

**Comment by Tohirjon-Odilov** [comment]
Javob yozvorilar

**Comment by Tohirjon-Odilov** [comment]
Kim kirsa ham

**Reply by Tohirjon-Odilov** [reply]
Savollar bo'lsa bemalol

**Reply by Tohirjon-Odilov** [reply]
Savollar bo'lsa bemallo

**Comment by 1lyosbek** [comment]
> ### Bosvoliroradi



> Nima gaplar

**Comment by 1lyosbek** [comment]
qanday olsak bo'ladi
